targets/targets.json

```diff
@@ -14793,6 +14793,61 @@
             "3701"
         ]
     },
+    "NUCLEO_G474RE": {
+        "inherits": [
+            "FAMILY_STM32"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
+            "MORPHO"
+        ],
+        "core": "Cortex-M4F",
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI",
+                "value": "USE_PLL_HSI",
+                "macro_name": "CLOCK_SOURCE"
+            },
+            "lpticker_lptim": {
+                "help": "This target supports LPTIM. Set value 1 to use LPTIM for LPTICKER, or 0 to use RTC wakeup timer",
+                "value": 1
+            },
+            "hse_value": {
+                "help": "HSE default value is 25MHz in HAL",
+                "value": "8000000",
```

Comment:
From board User Manual, I think HSE default value is 24MHz

Reply:
Updated to match reality, see 9ca09d283c553b40541e7064cd8cdb1b5bdb09bc